test/data.jl

```diff
@@ -190,8 +190,17 @@ module TestData
         @test size(dx) == (0, 3)
         @test names(dx) == [:variable, :value, :a]
 
-        stackdf(d1, :a)
         d1s = stackdf(d1, [:a, :b])
+        # TODO: add tests for Bool and Real for d1s columns 1 and 2 after deprecation of getindex
```

Comment:
Can you add these with `@test_broken`? That way we'll notice the failure when we remove the deprecations.

Reply:
added